src/actions/mover.ts

```diff
@@ -271,6 +279,7 @@ export class Mover {
     );
 
     this.unpatchIsDragging(workspace);
+    this.unpatchDragStrategy(info.block as BlockSvg);
```

Comment:
out of curiosity, is there a reason MoveInfo.block is a Block instead of a BlockSvg given you assume it'll always be the latter?

Reply:
Good point--and it becomes very obvious with these casts. Updated.